ADB_world/Worldmap_month/Data_frames.py

```diff
@@ -0,0 +1,128 @@
+########################################
+# IMPORTS #############################
+#######################################
+
+import json
+import pandas as pd
+import geopandas as gpd
+import os
+from shapely.geometry import LineString
+
+
+########################################
+# load data ############################
+########################################
+
+# Get the current directory
+current_directory = os.path.dirname(os.path.realpath(__file__))
```

Comment:
...same comment as in the `ADB_world/Worldmap_interactive/Main_interactive.py` file.